src/Geis.ScratchApp/Program.cs

```diff
@@ -0,0 +1,5 @@
+﻿using Geis;
```

Comment:
Not to be the old man "Are the kids wrong" but I'd prefer not using top level statements